src/app/project-notification/documents/upload/upload.component.html

```diff
@@ -56,10 +96,14 @@
       </li>
     </ul>
   </div>
-  <div class="text-right" >
+
+  <div class="text-right">
     <button class="btn btn-cancel" type="cancel" (click)="goBack()">Cancel</button>
     &nbsp;
-    <button class="btn btn-primary mr-1" [disabled]="!docNameExists() || !myForm.valid || docNameInvalid || documents?.length === 0"  type="submit" (click)="uploadDocuments()">Upload</button>
+    <button class="btn btn-primary mr-1"
+      [disabled]=" !myForm.valid || docNameInvalid || documents?.length === 0" type="submit"
```

Comment:
do we not need the `!docNameExists()` anymore?

Reply:
nope, it's being pulled right off the document now that the field is gone

Reply:
beauty